app/assets/stylesheets/main.scss

```diff
@@ -32,8 +31,99 @@ body {
       font-size: 25px;
     }
   }
+
+  ul{
```

Comment:
Try to make CSS changes that only affect YOUR code. If there's a reason for changing the style of ALL `ul` elements, then make sure everyone knows.